astronomer/starship/main.py

```diff
@@ -315,10 +315,11 @@ def dag_cutover_row_get(self, deployment: str, dag_id: str):
     def dag_cutover_row(
         self, deployment: str, dag_id: str, dest: str = "local", action: str = "init"
     ):
+        valid_actions = ["pause", "unpause", "migrate", "init"]
         if dest not in ["local", "astro"]:
             raise RuntimeError("dest must be 'local' or 'astro'")
-        if action not in ["pause", "unpause", "migrate", "init"]:
-            raise Exception("action must be 'pause', 'unpause', or 'migrate'")
+        if action not in valid_actions:
+            raise Exception(f"action must be in {valid_actions}")
```

Comment:
I think a better way to leverage this error message would be 

`            raise Exception(f"{action=} must be in {valid_actions=}")`

Which will show the current actiona nd allowed actions without making the person trying to run this package dive deep into the code base

Reply:
AWESOME! Haven't used that yet!

Reply:
Oh - hm, might not be available in all python we need to support due to `3.8`
https://docs.python.org/3/whatsnew/3.8.html#bpo-36817-whatsnew